tests/components/rfxtrx/test_switch.py

```diff
@@ -82,8 +84,7 @@ async def test_default_config(hass, rfxtrx):
         hass, "switch", {"switch": {"platform": "rfxtrx", "devices": {}}}
     )
     await hass.async_block_till_done()
-
-    assert 0 == len(rfxtrx_core.RFX_DEVICES)
+    assert hass.states.async_all() == []
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)